katportalclient/test/test_client.py

```diff
@@ -746,40 +864,40 @@ def test_sensor_history_single_sensor_with_value_ts(self):
         publish_messages.extend(sensor_history_pub_messages_json[sensor_name])
 
         self.mock_http_async_client().fetch.side_effect = mock_async_fetcher(
-            valid_response='{"result":"success"}',
+            valid_response=sensor_data3,
             invalid_response='error',
             starts_with=history_base_url,
             contains=sensor_name,
             publish_raw_messages=publish_messages,
             client_states=self._portal_client._sensor_history_states)
 
         samples = yield self._portal_client.sensor_history(
-            sensor_name, start_time_sec=0, end_time_sec=time.time(), include_value_ts=False)
+            sensor_name, start_time_sec=0, end_time_sec=time.time(), include_value_time=False)
         # expect exactly 4 samples
-        self.assertTrue(len(samples) == 4)
+        self.assertTrue(len(samples) == 3)
 
         # ensure time order is increasing
         time_sec = 0
         for sample in samples:
-            self.assertGreater(sample.timestamp, time_sec)
-            time_sec = sample.timestamp
+            self.assertGreater(sample.sample_time, time_sec)
+            time_sec = sample.sample_time
             # Ensure sample contains timestamp, value, status
-            self.assertEqual(len(sample), 3)
+            self.assertEqual(len(sample), 4)
```

Comment:
The first part of this test is with `include_value_time=False`, so the sample tuples must only have 3 items.  This line must be reverted.  Suggest using `sensor_data1` or `sensor_data2`, which don't have value timestamps for the mock query.

Second part of the test can use `sensor_data3`, when `include_value_time=True`.

I'm not sure why I tested for both with and without value timestamps in the same test.  Consider splitting this test into two: 
`test_sensor_history_single_sensor_without_value_time()`
`test_sensor_history_single_sensor_with_value_time()`.